CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 * [#60](https://github.com/nrepl/nrepl/issues/60): Implemented EDN transport.
 
+### Bugs fixed
+
+* [#152](https://github.com/nrepl/nrepl/issues/152): Bug fix for 'close' operation to kill session thread
```

Comment:
I guess a better wording would be "Kill session threads when closing sessions." (or something like this).